test/asm/sources/load.cc

```diff
@@ -0,0 +1,21 @@
+#include <eve/eve.hpp>
+
+#include "common.hpp"
+
```

Comment:
Do we need one file per function are can we aggregate them ?

Reply:
We don't. You can put them in whatever files you want (file names don't matter as well. The resulting asm will be split in files by function name, 
like I put `load` and `loadu` in one file.

I would like for a better way to generate all variations. If you habe suggestions, would be cool.
The question is to make compiler instantiate them

Reply:
It's tricky cause it depends on a lot of out-of-code knowledge.